dist/manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "PMMG Beautifier",
   "description": "Improve and tweak the interface elements of PrUn website",
-  "version": "0.4.1",
+  "version": "0.4.1.1",
```

Comment:
`0.5.0` please

Reply:
already deleted this file, didn't want any mismatch